launch/tier4_localization_launch/launch/util/util.launch.xml

```diff
@@ -13,11 +13,10 @@
 
   <!-- pose_initializer -->
   <group>
+    <let name="config_file" value="$(var pose_initializer_param_path)" if="$(eval &quot;'$(var run_mode)'=='online'&quot;)"/>
+    <let name="config_file" value="$(var pose_initializer_param_logging_simulator_path)" if="$(eval &quot;'$(var run_mode)'=='logging_simulation'&quot;)"/>
     <include file="$(find-pkg-share pose_initializer)/launch/pose_initializer.launch.xml">
-      <arg name="gnss_enabled" value="true"/>
-      <arg name="ndt_enabled" value="true"/>
-      <arg name="ekf_enabled" value="true"/>
-      <arg name="stop_check_enabled" value="true"/>
+      <arg name="config_file" value="$(var config_file)"/>
```

Comment:
Is it possible to have the default settings loaded in a yaml file and then override the parameters?

like this.
```
<arg name="config_file" value="$(var pose_initializer_param_path)"/>
<arg name="stop_check_enabled" value="false" if="$(eval &quot;'$(var run_mode)'=='logging_simulation'&quot;)/>
```

If it's possible, I think this would be better.

Reply:
@isamu-takagi What do you think?

Reply:
@YamatoAndo 
I think it's a good solution for logging simulator problems. Ideally it would be nice to be able to set it in a parameter file from autoware_launch repository, but that can be done in a separate PR.

Reply:
@isamu-takagi Thank you for the reply.
But after talking with Minoda-san, we decided to follow Takagi-san's suggestion since we want to write everything in parameter files.